dashboard/lib/pd/foorm/foorm_parser.rb

```diff
@@ -137,7 +137,11 @@ def self.get_friendly_rating_choices(question_data)
     def self.flatten_choices(choices)
       choices_obj = {}
       choices.each do |choice_hash|
-        choices_obj[choice_hash[:value]] = fill_question_placeholders(choice_hash[:text])
+        if choice_hash.class == Hash && choice_hash.key?(:value) && choice_hash.key?(:text)
+          choices_obj[choice_hash[:value]] = fill_question_placeholders(choice_hash[:text])
+        elsif choice_hash.class == String
+          choices_obj[choice_hash] = fill_question_placeholders(choice_hash)
```

Comment:
Should we emit a warning anywhere when we notice this?

Reply:
good question--should I add a honeybadger warning? Something else?

Reply:
It does seem that we use honeybadger for these kinds of alerts sometimes, with the idea that we will remedy the issue to try and drive instances to 0 each time it happens.  The only downside is reporting the issue every time we render a form, when we really just need to capture it once.  For that reason, it could make more sense to detect during seeding (and later, in the authoring UI itself too).  But with our current usage levels it doesn't seem unreasonable to just catch in honeybadger.

Reply:
added honeybadger alert. Let me know if the wording makes sense.